File: app/src/main/java/com/inspiredandroid/linuxcommandbibliotheca/ScriptChildrenActivity.java
package com.inspiredandroid.linuxcommandbibliotheca;

import android.os.Bundle;
import android.support.v7.widget.Toolbar;
import android.view.MenuItem;

/**
 * Created by Simon Schubert
 * <p/>
 * This Activity just holds the AboutFragment
 */
public class ScriptChildrenActivity extends BaseActivity {

    public final static String EXTRA_CATEGORY_ID = "EXTRA_CATEGORY_ID";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_scriptchildren);

        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        assert getSupportActionBar() != null;

        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        // int categoryId = getIntent().getIntExtra(EXTRA_CATEGORY_ID, ScriptGroupItem.GROUP_USER_GROUP);
        // setTitle(getDescriptionForCategory(categoryId));
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            case android.R.id.home:
                this.finish();
        }

        return (super.onOptionsItemSelected(item));
    }

}

File: app/src/main/java/com/inspiredandroid/linuxcommandbibliotheca/ScriptChildrenActivity.kt
package com.inspiredandroid.linuxcommandbibliotheca

import android.os.Bundle
import android.support.v7.widget.Toolbar
import android.view.MenuItem
import android.view.View

/**
 * Created by Simon Schubert
 *
 *
 * This Activity just holds the AboutFragment
 */
class ScriptChildrenActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_scriptchildren)

        val toolbar = findViewById<View>(R.id.toolbar) as Toolbar
        setSupportActionBar(toolbar)

        assert(supportActionBar != null)

        supportActionBar!!.setDisplayHomeAsUpEnabled(true)

        // int categoryId = getIntent().getIntExtra(EXTRA_CATEGORY_ID, ScriptGroupItem.GROUP_USER_GROUP);
        // setTitle(getDescriptionForCategory(categoryId));
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            android.R.id.home -> this.finish()
        }

        return super.onOptionsItemSelected(item)
    }

    companion object {

        val EXTRA_CATEGORY_ID = "EXTRA_CATEGORY_ID"
    }

}
